src/main/resources/alfio/db/PGSQL/V19_1.11.0__EXPAND_CONFIGURATION_VALUE.sql

```diff
@@ -16,6 +16,7 @@
 --
 
 alter table configuration alter column c_value SET DATA TYPE text;
+alter table configuration add column last_modified_date timestamp with time zone;
```

Comment:
you don't modify an existing sql migration file, additionally, the value last modified date is not set, so adding only the column is not helpful